lib/cucloud/ec2_utils.rb

```diff
@@ -50,8 +50,7 @@ def start_instance(instance)
     end
 
     # Set the name of the instance that will be displayed in the ec2 console
-    def rename_instance(instance, name)
```

Comment:
Should we just remove these empty methods?

Reply:
makes sense - doing that now